examples/existing-image/README.md

```diff
@@ -0,0 +1,114 @@
+# EXAMPLE: Deploying Terraform Enterprise in Active/Active mode with a custom image
+
+## About This Example
+
+This example functions as a reference for how to use this module to install 
+Terraform Enterprise with a custom image (AMI) in AWS.
+
+## Module Prerequisites
+
+As with the main version of this module, this example assumes the following
+resources already exist:
+
+- Valid DNS Zone managed in Route53
+- Valid AWS ACM certificate
+- Valid TFE license
+
+## How to Use This Module
+
+Refer to the root module's instructions for [setup instructions](../../README.md#How-to-Use-This-Module).
+
+- Any variables not defined in this example will use the default values of the
+root module, which defaults to a node count of 2, creating an Active/Active configuration.
+- Ensure account meets module pre-requisites from above.
+- Create a Terraform configuration that pulls in this module and specifies values
+  of the required variables (you may do this in a `terraform.tfvars` file):
+
+```hcl
+provider "aws" {}
+
```

Comment:
We should avoid including provider configuration in examples so that they are functional in as many contexts as possible.

```suggestion
```

Reply:
Whoops, meant to remove that. Thanks!